MapboxNavigation/UILabel.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+protocol Abridgeable: class {
+    var unabridgedText: String? { get set }
+    typealias AvailableBoundsHandler = () -> (CGRect)
+    var availableBounds: AvailableBoundsHandler! { get set }
+}
+
+extension Abridgeable where Self: UILabel {
+    func set(unabridgedText: String?) {
```

Comment:
Could this be a getter on `unabridgedText` instead of a separate method? If it does have to be a separate method, `setUnabridgedText(_:)` would be more forwards-compatible than `set(unabridgedText:)`. For example, in the future, if we need to add another string-typed setter, Swift won’t let us have both `set(unabridgedText:)` and `set(voiceIdentifier:)` on the same class.